asm-to-pil/src/vm_to_constrained.rs

```diff
@@ -128,19 +122,10 @@ struct VMConverter<T> {
     line_lookup: Vec<(String, String)>,
     /// Names of fixed columns that contain the rom.
     rom_constant_names: Vec<String>,
-    /// the maximum number of inputs in all functions
-    output_count: usize,
```

Comment:
Why don't we need this any more?

Reply:
It got moved somewhere else iirc, cleaner now